knative/natss-broker.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: natss
+#  namespace: {{ $.Release.Namespace }}
+spec:
+  channelTemplateSpec:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: NatssChannel
+#    kind: ClusterChannelProvisioner
+#    name: natss
```

Comment:
What is this?

Reply:
this is the kne broker that interfaces with the natss server